### DISCUSSION TITLE
Turning off an individual port on RPi 4

### DISCUSSION THREAD

**Discussion by Adrien-a0** [mvp/uhubctl#426] [General]
Hey, I'm a bit confused by the FAQ section on the RPi's. Firstly I would like to ask is it in fact possible to turn off the power to an individual usb port (or at least not all of them at once)? And secondly, if this is the case, how would one do this for e.g usb port 2? 

Thanks in advance for any help

**Comment by mvp** [comment]
No, all Raspberry Pi models can only turn off/on all 4 USB ports at once. For RPi4, you have to turn off all 4 ports:
```
uhubctl -l 1-1 -a 0
```
Not specifying `-p` means to turn off all ports here.

For older RPi models (2B, 3B, 3B+) you need to turn off port 2, it will actually remove power from all ports:
```
uhubctl -l 1-1 -p 2 -a 0
```
FAQ lists all commands that you should use https://github.com/mvp/uhubctl#raspberry-pi-turns-power-off-on-all-ports-not-just-the-one-i-specified
